R/model.R

```diff
@@ -212,17 +212,22 @@ model_emax <- function(
   )
   class(mod) <- c("dreamer_emax", "dreamer_continuous")
   attr(mod, "type") <- "EMAX"
+  rlang::inform("Note updated EMAX model parameterization", 
```

Comment:
Can we update this message to something like "EMAX model parameterization has changed in dreamer version 4.   Please review the new model parameterization and update prior hyperparameters accordingly."